website/docs/best-practices/how-we-mesh/mesh-3-structures.md

```diff
@@ -42,15 +48,37 @@ Horizontal splits separate your DAG based on source or domain. These splits are
 - **Pick one type of split and focus on that first**. If you have a hub-and-spoke team topology for example, handle breaking out the central platform project before you split the remainder into domains. Then if you need to break those domains up horizontally you can focus on that after the fact.
 - **DRY applies to underlying data, not just code.** Regardless of your strategy, you should not be sourcing the same rows and columns into multiple nodes. When working within a mesh pattern it becomes increasingly important that we don't duplicate logic or data.
 
+
+<Lightbox src="/img/best-practices/how-we-mesh/combined_splits.png" title="A simplified dbt DAG with two dotted lines representing both a vertical and horizontal split." />
+
+
 ## Determine your git strategy
 
 A multi-project architecture can exist in a single repo (monorepo) or as multiple projects, with each one being in their own repository (multi-repo).
 
 - If you're a **smaller team** looking primarily to speed up and simplify development, a **monorepo** is likely the right choice, but can become unwieldy as the number of projects, models and contributors grow.
 - If you’re a **larger team with multiple groups**, and need to decouple projects for security and enablement of different development styles and rhythms, a **multi-repo setup** is your best bet.
+
+## Projects, splits, and teams
+
+Since the laumch of dbt Mesh, the most common pattern we've seen is one where **projects are 1:1 aligned to teams**, and each project is its own codebase in its own repository. This isn’t a hard-and-fast rule: Some organizations want multiple teams working out of a monorepo. Some teams own multiple domains that feel awkward to keep combined.
+
+Users may need to contribute models across multiple projects. This is fine! There will be some more friction to do this, versus the monorepo — but this is **useful** friction, especially if upstreaming a change from a “spoke” to a “hub.” This should be treated like making an API change, one that the other team will be living with for some time to come. You should be concerned if your teammates find they need to make a coordinated change across multiple projects very frequently (every week), or as a key prerequisite for ~20%+ of their work.
+
+### Tips and tricks
+
+In the next page, we'll go through a more in depth example of how to split a monolithic project into multiple projects. Here are some tips to get you started when starting to think about using the splitting methods listed above on your own projects:
+
+1. Start by drawing a diagram of your teams doing data work. Map each team to a single dbt project. If you already have an existing monolithic project, and you’re onboarding *net-new teams,* this could be as simple as declaring the existing project as your “Hub” and creating new “Spoke” sandbox projects for each of those teams.
```

Comment:
```suggestion
1. Start by drawing a diagram of your teams doing data work. Map each team to a single dbt project. If you already have an existing monolithic project, and you’re onboarding _net-new teams,_ this could be as simple as declaring the existing project as your “hub” and creating new “spoke” sandbox projects for each team.
```